mail_tracking_mailgun/models/res_partner.py

```diff
@@ -192,14 +188,6 @@ def _autocheck_partner_email(self):
         for partner in self:
             partner.with_context(mailgun_auto_check=True).check_email_validity()
 
-    @api.model
-    def create(self, vals):
```

Comment:
Why this has been removed?

Reply:
This couldn't be working with the previous code so I decided to just drop it as I assume none is using it

Reply:
Then remove all the references also in documentation.

Reply:
Done